.github/workflows/ci.yml

```diff
@@ -53,6 +53,7 @@ jobs:
         run: pnpm test
 
       - name: Typecheck
+        if: matrix.node_version != '18.20.7'
```

Comment:
Error log: https://github.com/unocss/unocss/actions/runs/15613165647/job/43979000089#step:9:12

I will temporarily abandon support for lower versions of typecheck, hoping it will be fixed in the future.